ush/generate_FV3LAM_wflow.sh

```diff
@@ -610,6 +621,7 @@ ${EXPTDIR}/run_rocoto.sh"
 
 echo "#!/bin/bash" > ${EXPTDIR}/run_rocoto.sh
 if [[ "${MACHINE,,}" == "wcoss2" ]] ; then
+  rocoto_ver=1.3.5
```

Comment:
I think the `rocoto_ver` env variable should be available before line 625. 

@chan-hoo Could you confirm whether we need to hardwire a version information in line 624? Thanks!

Reply:
To meet the NCO standards, the module version should be defined in `rrfs-workflow/versions/run.ver` on WCOSS2.